src/test/resources/META-INF/frontend/styles/extended-login-styles-v25.css

```
@@ -0,0 +1,40 @@
+/*-
```

Stylelint (17.4.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)